Makefile

```diff
@@ -29,6 +29,8 @@ gsolver_solid_qdht : CPPFLAGS = -O3 -g -DKELDYSH_IONIZATION -D_EXTMATH_DOUBLE -D
 gsolver_solid_ux : CPPFLAGS = -O3 -g -DKELDYSH_IONIZATION -D_EXTMATH_DOUBLE -D_FILE_OFFSET_BITS=64 -D_ODE_RK4 -DPLASMA_DISPERSION -DFFTW_FLAG=FFTW_MEASURE -DTRANSVERSE_DIMENSIONS=1
 gsolver_solid_1d : CPPFLAGS = -O3 -g -DKELDYSH_IONIZATION -D_EXTMATH_DOUBLE -D_FILE_OFFSET_BITS=64 -D_ODE_RK4 -DPLASMA_DISPERSION -DFFTW_FLAG=FFTW_MEASURE -DNO_DIFFRACTION
 gsolver_solid : CPPFLAGS = -O3 -g -DKELDYSH_IONIZATION -D_EXTMATH_DOUBLE -D_FILE_OFFSET_BITS=64 -D_ODE_RK4 -DPLASMA_DISPERSION -DFFTW_FLAG=FFTW_MEASURE -DTRANSVERSE_DIMENSIONS=2
+gsolver_solid_qdht : CPPFLAGS = -O3 -g -DKELDYSH_IONIZATION -D_EXTMATH_DOUBLE -D_FILE_OFFSET_BITS=64 -D_ODE_RK4 -DPLASMA_DISPERSION -DFFTW_FLAG=FFTW_ESTIMATE -DTRANSVERSE_DIMENSIONS=1 -DQDHT -D_UNIAXIAL
```

Comment:
Why change the flags?

Reply:
Which one? If you are talking about fftw flags - to make code start faster (measure vs estimate don't matter much in short term, to my experience). The rest of them - to make a axisymmetric solver with Keldysh ionization